query-engine/connectors/sql-query-connector/src/query_builder/read.rs

```diff
@@ -199,11 +201,15 @@ pub(crate) fn aggregate(
             }),
 
             AggregationSelection::Min(fields) => fields.iter().fold(select, |select, next_field| {
-                select.value(min(Column::from(next_field.db_name().to_owned())))
+                select.value(min(Column::from(next_field.db_name().to_owned())
+                    .set_is_enum(next_field.type_identifier().is_enum())
+                    .set_is_selected(true)))
             }),
```

Comment:
These outer columns (see comment above) are now set as enum (if they are) and selected

Reply:
Thanks for these clarifying comments